<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>System_config</class>
 <widget class="QWidget" name="System_config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>System_config</string>
  </property>
  <widget class="QTabWidget" name="tabWidget_config">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>411</width>
     <height>291</height>
    </rect>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::West</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="camera_config">
    <attribute name="title">
     <string>camera</string>
    </attribute>
    <widget class="QComboBox" name="Camera_parameter">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTipDuration">
      <number>-1</number>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="Camera_parameter_value">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="model_config">
    <attribute name="title">
     <string>model</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="3">
      <widget class="QLineEdit" name="lineEdit_position_accuracy"/>
     </item>
     <item row="2" column="4">
      <layout class="QHBoxLayout" name="horizontalLayout_3"/>
     </item>
     <item row="4" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_10"/>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="lineEdit_size_accuracy"/>
     </item>
     <item row="1" column="4">
      <layout class="QHBoxLayout" name="horizontalLayout_2"/>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>position accuracy</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <layout class="QHBoxLayout" name="horizontalLayout_4"/>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="lineEdit_max_position"/>
     </item>
     <item row="4" column="4">
      <layout class="QHBoxLayout" name="horizontalLayout_5"/>
     </item>
     <item row="0" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_6"/>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>max position</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLineEdit" name="lineEdit_scaling_factor"/>
     </item>
     <item row="2" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_8"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>initial position</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>size accuracy</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>scaling factor</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="lineEdit_initial_position"/>
     </item>
     <item row="3" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_9"/>
     </item>
     <item row="1" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_7"/>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_11"/>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_12"/>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_13"/>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_14"/>
     </item>
     <item row="4" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_15"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="Btn_apply">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
